### PR TITLE
Adds a blog post for forecasting timeseries data in VSCode Jupyter Notebooks.

### PR DESCRIPTION
**Pull request by KieranBrantnerMagee**
Leverages blob storage (and logs derived from app insights) as persistance to demonstrate ARIMA prediction and result persistance.

The actual notebook can be found within the public URL within the blog itself; PRing notebooks has always been an "interesting" concept so it seems fitting to review the markdown generated off of the notebook instead.

### CODE REVIEW

File: _posts/2020-04-24-forecasting-blob-vscode.md [hunk 1]
```diff
@@ -0,0 +1,311 @@
+
+# Forecasting service scale-out using Azure Storage and VSCode Jupyter Notebooks #
+
+As you may or may not know, VSCode has been busy enabling Jupyter Notebook functionality within the editor, which to those of us with our fingers in data analytics, is quite a nice boon to simplify our dev workflows and environments.
```
Comment (adrianhall) [inline _posts/2020-04-24-forecasting-blob-vscode.md:4]: Remove initial part "As you may or may not know".  It's useless words.  Link to the Jupyter notebook functionality plugin

File: _posts/2020-04-24-forecasting-blob-vscode.md [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+
+Towards that effort, we hope to take this opportunity to walk through some interesting end-to-end examples of leveraging this tooling alongside our Azure capabilities (Blob storage, in this case).
+
+For this writeup, we'll demonstrate how to leverage App-Insights data (or really, any ad-hoc system-scale-related-telemetry) flowing into Azure Blob, to be consumed via experimentation in VSCode Jupyter for the purpose of designing a predictive model to anticipate service scale up/down, a common task for optimizing cloud spend and anticipating scale requirements.
```
Comment (adrianhall) [inline _posts/2020-04-24-forecasting-blob-vscode.md:8]: Azure Blob Storage.  Use the full name.

File: _posts/2020-04-24-forecasting-blob-vscode.md [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+
+<p align="center">
+    <img src="https://camo.githubusercontent.com/1acea3f1c347974d095b303ee4e4d8b33cc4901e/687474703a2f2f616c6b616c696e652d6d6c2e636f6d2f696d672f7374617469632f706d646172696d615f726561646d655f6578616d706c65312e706e67">
```
Comment (adrianhall) [inline _posts/2020-04-24-forecasting-blob-vscode.md:11]: Move the image into the images section of the website and link as a picture.

**Comment by KieranBrantnerMagee** [comment]
Closing this in leiu of the properly formatted version [here.](https://github.com/Azure/azure-sdk/pull/1275) (Thanks Adrian++)

**Pull request closed without merging**